### ISSUE TITLE
Nesting `@book`s?  

### ISSUE THREAD

**Issue by RandomDSdevel** [zyedidia/Literate#17]
@zyedidia:  Could you make it so that I can nest `@book`s if I can't do so already?  What I'd like to do is have a top-level `@book`-directive file containing the top-level table of contents, then have `lit` pull in any tables of contents it finds in `@book`-directive files linked to from this top-level one imported in to that latter one, allowing me to string together nested tables of contents.  For example:  

In the top-level `@book`-directive file:  

> ```
> ⋮
>
> @s Table of Contents
>
> [Reference to another `@book` directive file relative to this one](link-target)
> ```

In the referenced `@book`-directive file:  

> ```
> ⋮
>
> @s Table of Contents
>
> [Reference to another `@book` directive file relative to this one](link-target)
> ```

Desired result in top-level `@book`-directive file:  

> ```
> ⋮
>
> # Table of Contents
>
> 
>   
> ```

Basically, I want to be able to string tables of contents together like _that_.  This might be related to #7, methinks?  It might be useful to do something similar with ToCs outside of `@book`-directive files if those ever exist.

**Comment by RandomDSdevel** [comment]
In the meantime, perhaps I should just go learn LaTeX…

**Comment by zyedidia** [comment]
I'm sorry but it's unlikely I'll be able to implement something like this mostly because it has been a few years since I actively worked on this project and I don't have very much time at the moment.

**Comment by RandomDSdevel** [comment]
That's fine; I can use something else like LaTeX or one of the other Markdown literate-programming document systems available here on GitHub.  Good luck with your other project(s.)
